### PR TITLE
Add multiplex support

### PR DESCRIPTION
**Pull request by DmitryTsepelev**

### CODE REVIEW

File: lib/graphql/persisted_queries.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ module GraphQL
   module PersistedQueries
     def self.use(schema_defn, options)
```
Comment (JanStevens) [inline lib/graphql/persisted_queries.rb]: I now receive following error
```
/lib/graphql/persisted_queries.rb:13:in `use': wrong number of arguments (given 1, expected 2) (ArgumentError)
	from /ruby/2.6.5/lib/ruby/gems/2.6.0/gems/graphql-1.10.3/lib/graphql/schema.rb:863:in `use'
	from app/graphql/crew_schema.rb:16:in `<class:CrewSchema>'
```

This probably has to do with the following

https://github.com/rmosolgo/graphql-ruby/blob/120ec5e5fd6c9d485f57479cb3861ca1c4bb4a84/lib/graphql/schema.rb#L859-L866

I suspect that your use statement should take `options` as optional so it becomes

```ruby
def self.use(schema_defn, options = {})
```

Comment (DmitryTsepelev) [reply lib/graphql/persisted_queries.rb]: Good point, pushed the fix

**Comment by JanStevens** [comment]
Ok that seemed to work just fine! A couple of screenshots that show that in batch we return the right responses for the batched queries and then the request is done again with the query and the sha256

Screenshot 2020-02-21 at 09 43 32
Screenshot 2020-02-21 at 09 43 23
Screenshot 2020-02-21 at 09 43 17

**Comment by DmitryTsepelev** [comment]
If the batch is sent and _some_ of the queries are "not persisted", then BatchLink will figure that out and make a new request with only required queries, right?

**Comment by JanStevens** [comment]
> If the batch is sent and _some_ of the queries are "not persisted", then BatchLink will figure that out and make a new request with only required queries, right?

Testing that one out now, but I don't think I can easily do that with the in memory cache, going to need redis so I can delete one key on a page where I know I'm batching queries

**Comment by JanStevens** [comment]
Cool it seems it does work that way! So I deleted one key from a batch query for 3 queries, this results in one query not being found.

Screenshot 2020-02-21 at 09 57 06

The next request is just for the one query not being found instead of all of them
Screenshot 2020-02-21 at 09 57 23

**Comment by DmitryTsepelev** [comment]
Cool cool, I'll add some more tests and docs and roll out a new version, thank you!

**Pull request merged**